### ISSUE TITLE
[Bug]: Downloading JASP

### ISSUE THREAD

**Issue by Georgina514** [jasp-stats/jasp-issues#2323]
### JASP Version

0.18.0

### JASP Module

Other

### What OS are you seeing the problem on?

Windows 11

### Bug Description

Attempting to download JASP onto multiple student laptops and getting the error message of the site can't be reached. 
Is there a known issue at the moment with downloading JASP and if so, when will students be able to download this?

### Expected Behaviour

Download should of started.

### Steps to Reproduce

1. Click download for Windows then click here if download doesn't start, and nothing happens.
2.
3.
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by boutinb** [comment]
It works now.

**Comment by Ghassan-Okour** [comment]
> ### JASP Version
> 0.18.0
> 
> ### Commit ID
> _No response_
> 
> ### JASP Module
> Other
> 
> ### What analysis are you seeing the problem on?
> _No response_
> 
> ### What OS are you seeing the problem on?
> Windows 11
> 
> ### Bug Description
> Attempting to download JASP onto multiple student laptops and getting the error message of the site can't be reached. Is there a known issue at the moment with downloading JASP and if so, when will students be able to download this?
> 
> ### Expected Behaviour
> Download should of started.
> 
> ### Steps to Reproduce
> 1. Click download for Windows then click here if download doesn't start, and nothing happens.
> 
> ...
> 
> ### Log (if any)
> _No response_
> 
> ### Final Checklist
> * [x]  I have included a screenshot showcasing the issue, if possible.
> * [x]  I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
> * [x]  I have accurately described the bug, and steps to reproduce it.

Yes; I can not download both (the latest version and the latest flight of nighty builds) for Jasp

**Comment by EJWagenmakers** [comment]
This was a temporary glitch that the website company has resolved. It should work again. Please re-open if this is still an issue.

**Comment by Ghassan-Okour** [comment]
> This was a temporary glitch that the website company has resolved. It should work again. Please re-open if this is still an issue.

Yes; now downloading is going...
